.github/scripts/sync_labels.py

```diff
@@ -51,7 +51,7 @@ def get_linked_issues_based_on_pr_body(repo, number):
     pattern = fr'Fixes:? (?:#|{repo}#|https://github.com/{repo}/issues/)(\d+)'
     matches = re.findall(pattern, pr.body)
     if not matches:
-        raise RuntimeError("No regex matches found in the body!")
+        return []
```

Comment:
in https://github.com/scylladb/scylladb/pull/17920 I am already addressing those failures. This failure means we will not sync between Issues and PR since there is no connection between them. People must know that. removing the `RuntimeError` will make the script pass without doing anything. Then people start asking why it's not updated

Reply:
cool. i am closing this PR then.